src/test/java/de/tum/in/www1/artemis/tutorialgroups/AbstractTutorialGroupIntegrationTest.java

```diff
@@ -100,15 +107,43 @@ abstract class AbstractTutorialGroupIntegrationTest extends AbstractSpringIntegr
 
     Integer defaultSessionEndHour = 12;
 
-    LocalDate firstAugustMonday = LocalDate.of(2022, 8, 1);
+    final static LocalDate FIRST_AUGUST_MONDAY = LocalDate.of(2022, 8, 1);
 
-    LocalDate secondAugustMonday = LocalDate.of(2022, 8, 8);
+    final static LocalDate SECOND_AUGUST_MONDAY = LocalDate.of(2022, 8, 8);
 
-    LocalDate thirdAugustMonday = LocalDate.of(2022, 8, 15);
+    final static LocalDate THIRD_AUGUST_MONDAY = LocalDate.of(2022, 8, 15);
 
-    LocalDate fourthAugustMonday = LocalDate.of(2022, 8, 22);
+    final static LocalDate FOURTH_AUGUST_MONDAY = LocalDate.of(2022, 8, 22);
 
-    LocalDate firstSeptemberMonday = LocalDate.of(2022, 9, 5);
+    final static LocalDate FIRST_SEPTEMBER_MONDAY = LocalDate.of(2022, 9, 5);
+
+    final static LocalDateTime FIRST_AUGUST_MONDAY_00_00 = LocalDateTime.of(2022, 8, 1, 0, 0, 0);
```

Comment:
Although the naming is consistent, the arguments are not (5 vs. 6) is there a reason? If so document it. Otherwise, unify them.